hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskWithLegacy.java

```diff
@@ -206,6 +206,22 @@ public boolean processWithLegacy(OMUpdateEventBatch events) {
     return true;
   }
 
+  /**
+   * Checks if the object passed is an instance of `OmKeyInfo`.
+   *
+   * @param obj The object to check.
+   * @return True if the object is an instance of `OmKeyInfo`, false otherwise.
+   */
+  private boolean isOmKeyInfo(Object obj) {
```

Comment:
Can we move this method at common place ? We may not need this here or in any class if we add a logic in org.apache.hadoop.ozone.recon.tasks.OMDBUpdatesHandler#processEvent